xapian-applications/omega/docs/omegascript.rst

```diff
@@ -1018,8 +1018,9 @@ $if{COND[,THEN[,ELSE]]}
 
         The ability to omit ``THEN`` was added in Omega 1.4.15.
 
-$include{FILE}
-	include another OmegaScript file
+$include{FILE,[FAILOVER_SCRIPT]}
+        include another OmegaScript file ``FILE``, runs optional
+        OmegaScript ``FAILOVER_SCRIPT`` on failure.
```

Comment:
The `,` should be omitted when the second argument is, so that should be inside the `[`...`]` too - `$include{non_existent.template,}` will quietly expand to nothing!

Reply:
I think it would be clearer to word this like the docs for `$if` above - "runs" is a bit vague, whereas "evaluates and returns" makes it clear that the second argument is evaluated and the result of that returned.

I worry `FAILOVER_SCRIPT` might be misunderstood to mean a script file (i.e. another file to include instead of `FILE`.  Perhaps `$include{FILE[,FALLBACK]}`?

We should almost make it clear what "failure" means.  I think (see below) that it probably should only be a failure to open the file to be included.